databricks/koalas/base.py

```diff
@@ -391,7 +391,7 @@ def hasnans(self):
         sdf = self._internal.spark_frame
         scol = self.spark.column
 
-        return sdf.select(F.max(scol.isNull() | F.isnan(scol))).collect()[0][0]
```

Comment:
Maybe I think we don't need `F.isnan(scol)` because `np.nan` is casted to `null` when the Koalas DataFrame is casted to PySpark'.

Reply:
@itholic there can be `NaN` from Spark DataFrame directly. For example,

```python
import pyspark
from databricks import koalas as ks
spark = pyspark.sql.SparkSession.builder.getOrCreate()
df = spark.createDataFrame([[1.0, float('nan')]])
df.show()
kdf = df.to_koalas()
```

Does it work with the `kdf` too?

Reply:
@HyukjinKwon Ah, I got it. We should separate the condition.

`ks.DataFrame` has no attribute `hasnans` just same as `pd.DataFrame`